Apps/Database/src/Models/BetaRequest.cs

```diff
@@ -0,0 +1,33 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Database.Models
+{
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
+    using HealthGateway.Database.Constants;
+
+    /// <summary>
+    /// Represents a request to access the service in beta mode.
+    /// TODO: Finish DB model
+    /// </summary>
+    public class BetaRequest : AuditableEntity
+    {
+        // TODO:
```

Comment:
Why do you have an empty todo?

Reply:
Because doing the model and DB update is part of 7833